assets/js/modules/ads/components/setup/SetupMainPAX.js

```diff
@@ -79,16 +93,20 @@ export default function SetupMainPAX( { finishSetup } ) {
 	const { setPaxConversionID, setExtCustomerID, submitChanges } =
 		useDispatch( MODULES_ADS );
 
-	const onCompleteSetup = useCallback( async () => {
-		if ( ! paxApp ) {
+	const onLaunch = ( app ) => {
+		paxApp.current = app;
+	};
+
+	const onCampaignCreated = useCallbackOne( async () => {
+		if ( ! paxApp.current ) {
 			return;
 		}
 
 		/* eslint-disable sitekit/acronym-case */
 		// Disabling rule because function and property names
 		// are expected in current format by PAX API.
 		const { accountService, conversionTrackingIdService } =
-			paxApp.getServices();
+			paxApp.current.g;
```

Comment:
Apparently there is no `getServices()` anymore in passed `app` object, it became `g`

Reply:
Solved by changing integrator version to the `latest`